### ISSUE TITLE
Review acknowledgments and editors list

### ISSUE THREAD

**Issue by wseltzer** [w3c/webauthn#706]
Let's review the acknowledgments to get them in shape for CR. 

Shall we alphabetize the list of editors?

**Comment by jcjones** [comment]
+1 to alphabetize the editors.

I also would propose we move some of the editors who've not been involved recently to 'former editors'.

**Comment by equalsJeffH** [comment]
@levangongPayPal has already told me he's fine with being listed as "contributor".

**Comment by selfissued** [comment]
We should add Akshay as a current editor.

**Comment by nadalin** [comment]
@selfissued @jcjones @equalsJeffH I will make a call for additional editors on next weeks call so we are all in agreement

**Comment by equalsJeffH** [comment]
Summary: In case it is helpful, I've extracted the names/userids of all webauthn committers from the webauthn git repo. Here is the list along with a commit count for each (note: some commits by various folks were to various ancilliary items in the repo, such as README.md, the travis & bikeshed stuff, etc):
```
> git hist | grep -Fof webauthn-committer-names.txt | sort -f | uniq -c
   1 [Ackermann Yuriy]
   1 [Adam Langley]
   1 [Akshay Kumar]
  16 [Alexei Czeskis]
  26 [Angelo Liao]
  41 [AngeloKai]
  12 [balfanz]
   1 [bifurcation]   (Richard Barnes)
   9 [Christiaan Brand]
   8 [christiaanbrand]
   6 [Dirk Balfanz]
  63 [Emil Lundberg]
   8 [gmandyam]
   4 [Harry Halpin]
  75 [J.C. Jones]
   5 [James 'J.C.' Jones]
   1 [James Barclay]
  41 [=JeffH]
  72 [JeffH]
  22 [Jeffrey Yasskin]
   8 [Johan Verrept]
   2 [kpaulh]
   1 [levangongPayPal]
  32 [Mike Jones]
  44 [Mike West]
   1 [Philip Jägenstedt]
   1 [Rahul Ghosh]
  22 [rlin1]
  11 [Rolf Lindemann]
   1 [Samuel Weiler]
   1 [samweiler]
  46 [Vijay Bharadwaj]
   1 [Wally Jones]
  23 [Wendy Seltzer]
   1 [wseltzer]
   2 [ynojima]
```
tl;dr: I did an initial extraction from the git hist output of any line containing a substring matching "\[.*]\]" using:
`git hist | egrep -o "\[.*\]" | sort -f | uniq -ic`

..which output all committer names/userids plus various spurious noise since some commit comments have text containing square brackets.  I extracted the committer names/userids from that output by hand, and created the `webauthn-committer-names.txt` file (below), which i subsequently used to produce the above "clean" committer list.  
```
> cat webauthn-committer-names.txt
[J.C. Jones]
[Emil Lundberg]
[=JeffH]
[Jeffrey Yasskin]
[Ackermann Yuriy]
[Adam Langley]
[Vijay Bharadwaj]
[Akshay Kumar]
[Alexei Czeskis]
[Angelo Liao]
[AngeloKai]
[balfanz]
[bifurcation]
[Christiaan Brand]
[christiaanbrand]
[Dirk Balfanz]
[JeffH]
[gmandyam]
[Harry Halpin]
[James 'J.C.' Jones]
[James Barclay]
[Johan Verrept]
[kpaulh]
[levangongPayPal]
[Mike Jones]
[Mike West]
[Philip Jägenstedt]
[Rahul Ghosh]
[rlin1]
[Rolf Lindemann]
[Samuel Weiler]
[samweiler]
[Wally Jones]
[Wendy Seltzer]
[wseltzer]
[ynojima]
```

**Comment by equalsJeffH** [comment]
In terms of acknowledging folks who've provided feedback on the spec and/or submitted issues, I wonder if there is a means to do something similar to the committer info extraction in https://github.com/w3c/webauthn/issues/706#issuecomment-349718182, and generate a list of names/userids that have submitted issues?
